### DISCUSSION TITLE
MS SQL server (2019) connection

### DISCUSSION THREAD

**Discussion by Boyangqi** [apache/superset#23119] [Q&A / Help]
Hello!

I'm new to the apache superset, and I have a question about its connection to MS SQL server(2019). I wonder if the superset can be connected to it directly. I found controversial answers about it. Some said that it doesn't support the 2019 version and a workaround would be to replicate the database to PostgreSQL. But others said it did connect to it. Has anyone done that before? 

Thank you!

**Comment by sfirke** [comment]
Do you mean as a data source or as the metadata backend for storing Superset's data about users, charts, dashboards?

You almost certainly can query it as a data source.  But SQL Server is not supported as a metadata/backend store.  You could _try_ it and maybe it would work, maybe it wouldn't, but if it fails it would wreck everything.
